yarn-project/circuits.js/src/abis/__snapshots__/abis.test.ts.snap

```diff
@@ -152,46 +152,6 @@ exports[`abis wasm bindings hash constructor info 2 args 1`] = `
 }
 `;
 
-exports[`abis wasm bindings hashes VK 1`] = `
```

Comment:
This was removed?

Reply:
I think we re-generated the snapshot and this one was removed because the corresponding test is skipped. AT least, this is my understanding, namely from abis.test.ts:
`// TODO: This test fails on CI since build-system is not updating the latest circuits wasm
  // We may need to wait until we bump to the next commit to see if it picks up the change
  it.skip('hashes VK', async () => {
    const vk = makeVerificationKey();
    const res = await hashVK(wasm, vk.toBuffer());
    expect(res).toMatchSnapshot();
  });`